AUTHORS

```diff
@@ -378,7 +377,7 @@ Eric Miller <emiller42@gmail.com>
 Venkata Ramana <idlike2dream@gmail.com>
 Cody Herbst <cyherbst@gmail.com>
 Nishith Shah <nishithshah.2211@gmail.com>
-AMiT Kumar <dtu.amit@gmail.com>
+Amit Kumar <dtu.amit@gmail.com>
```

Comment:
Yes! [here](https://github.com/aktech) is the link to his GitHub profile. From here I figured out that "Amit" would be more appropriate to keep.